src/serve/dashboard/views/utils/validators.ts

```diff
@@ -0,0 +1,25 @@
+// Simple validators for form validation
+export const required = (value: unknown): boolean => {
+  if (value === null || value === undefined) return false
+  if (typeof value === 'string') return value.trim().length > 0
+  if (Array.isArray(value)) return value.length > 0
+  return true
+}
+
+export const email = (value: string): boolean => {
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
```

Comment:
This regex is quite different from what vuelidate has, and still both this and the Vuelidate one, as well as the one browsers use for `<input type="email">` are technically wrong (`alice@example`, `alice@[::1]`, `alice."some text".smith@example.com` are all valid email addresses). That said, this regex does catch most email addresses in use.